'use strict';

const retry = (f, n) => n > 0 ? f().catch(_ => retry(f, n-1)) : f();

module.exports = retry;
